placeholderkv.conf

```diff
@@ -327,19 +325,19 @@ daemonize no
 #
 # supervised auto
 
-# If a pid file is specified, Redis writes it where specified at startup
+# If a pid file is specified, the server writes it where specified at startup
 # and removes it at exit.
 #
 # When the server runs non daemonized, no pid file is created if none is
 # specified in the configuration. When the server is daemonized, the pid file
-# is used even if not specified, defaulting to "/var/run/redis.pid".
+# is used even if not specified, defaulting to "/var/run/server.pid".
 #
-# Creating a pid file is best effort: if Redis is not able to create it
+# Creating a pid file is best effort: if the server is not able to create it
 # nothing bad happens, the server will start and run normally.
 #
-# Note that on modern Linux systems "/run/redis.pid" is more conforming
+# Note that on modern Linux systems "/run/server.pid" is more conforming
 # and should be used instead.
-pidfile /var/run/redis_6379.pid
+pidfile /var/run/server_6379.pid
```

Comment:
Here too. We may want to pick something more unique here.

Reply:
With the port number in it, I thought it might be fine. Do you still find it as a risk? We can follow what I mentioned for the above comment.

Reply:
Ok, you're right, with the port number it's collision safe. And it's just a default in the example config file.